src/hooks/ensName.ts

```diff
@@ -60,7 +60,7 @@ export function useEnsName(address: string | undefined) {
     const now = new Date().valueOf()
 
     async function loadEnsName() {
-      if (!address) return
+      if (!address || !isAddress(address)) return
```

Comment:
small thing i noticed that was creating errors while typing a partial address. don't think this breaks anything.

Reply:
(i'm just passing by, non-blocking) => could consolidate line 63 + 65 with 

```suggestion
    const normalizedAddress = safeGetAddress(address)
     if (!normalizedAddress) return
```